CryptoTechReminderSystem.Test/UseCase/RemindDeveloperTests.cs

```diff
@@ -38,7 +39,7 @@ public void CanRemindDeveloper2()
         {
             var spy = new SlackGatewaySpy();
             var remindDeveloper = new SendReminder(spy);
-            var text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
+            const string text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
```

Comment:
Extracted into a `private const string` as it is used multiple times in this test suite.

Reply:
Solved in https://github.com/chubberlisk/crypto_tech/pull/35/commits/a5de69c15e5871e50fa7be6312071e9d119792f7.